nx/test/nx/tensor_test.exs

```diff
@@ -129,4 +131,56 @@ defmodule Nx.TensorTest do
       end
     end
   end
+
+  describe "inspect" do
```

Comment:
@a-alhusaini This PR will introduce a paliative solution for this. The printing format (and maybe the actual option) will change in the future, but it should solve your problem in the short-term.

The changes below contain some examples on the inspected tensors in the "prints with configured precision" test.

For usage in IEx, you can use `IEx.configure(inspect: [custom_options: [nx_precision: 5]])` for example.